config/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
 	"singleQuote": true,
 	"trailingComma": "all",
```

Comment:
This is now the default, and also it got renamed to `es5` a while back and support for `all` got dropped in 2.0

Reply:
`es5` is the default as of 2.0, `always`/`all` are still valid configs which behave differently. See https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker (also I tested it).

FWIW I actually prefer the `es5` format and was considering making an issue suggesting we move that.

Reply:
the `all` -> `es5` changes removes command like this

```diff
     logger.info(
-       `AUTH FAIL: API Authentication failed for ${req.body.username}`,
+       `AUTH FAIL: API Authentication failed for ${req.body.username}`
    );
```

Reply:
Ah I thought they were the same, I guess I got confused with the other options. In that case I'd stick with `all`, and it sounds like it might also happen in prettier as well in time as well:
> There is a possibility that the default value will change to always (meaning even more trailing commas) in a future major version of Prettier as the JavaScript ecosystem further matures.